e2e/fixtures/css/main.css

```diff
@@ -0,0 +1,8 @@
+/* @OctoLinkerResolve(<root>/css/import.css) */
+@import 'import.css';
+
+/* @OctoLinkerResolve(<root>/css/import.css) */
+@import "import.css";
```

Comment:
I think we can remove one of the test, since different quote marks are already covered in the RegExp test https://github.com/OctoLinker/OctoLinker/blob/main/packages/helper-grammar-regex-collection/test.js#L309-L310

Reply:
I switched them all to single quotes since that's what we use elsewhere, and I removed the duplicate test.